### PR TITLE
add translation

### PR DESCRIPTION
**Pull request by elec3c**
## Description of improvements

Any specific reason for this change? If a recent change in the MODX core caused the change, please include a link to the relevant pull request. 

## Affected versions

Please make sure your pull request targets the **lowest** affected version. Mark other relevant versions with an X below.

- [ ] 2.x (current)
- [ ] 3.x

## Relevant issues

If available, provide the link or ID of relevant issues so that they can be closed once your improvement has been merged.

### CODE REVIEW

**Review by Ibochkarev — CHANGES_REQUESTED** [review]
Пожалуйста, удалите, не переведенный кусок текста.

File: ru/building-sites/client-proofing/security/policies/policytemplates.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Элемент определяет дополнительные разрешения, связанные с ACL Элемента.
+
+### MediaSourceTemplate
+
+Разрешения, связанные с доступом к источникам файлов.
+
+### ObjectTemplate
+
+Общие разрешения xPDOObject использующиеся для любого modAccessibleObject.
```
Comment (Ibochkarev) [inline ru/building-sites/client-proofing/security/policies/policytemplates.md]: Название классов заключить в ` кавычки

File: ru/building-sites/client-proofing/security/policies/permissions/resource-policy.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+## Ресурсная политика
+
+This policy is packaged into MODX and is given to users on any context who want to have basic object access to content. The Permissions are generic and apply across all MODX objects.
+Эта политика соержется в MODX и предоставляется пользователям, которые хотят иметь базовый объектный доступ к контенту, в любом контексте. Разрешения являются общими и применяются ко всем объектам MODX.
+
+## Разрешения по умолчанию
+
+| Name |
```
Comment (Ibochkarev) [inline ru/building-sites/client-proofing/security/policies/permissions/resource-policy.md]: Перевести заголовок таблицы

**Comment by elec3c** [comment]
Внёс правки

File: ru/building-sites/client-proofing/security/policies/policytemplates.md (hunk 1 above)
Comment (elec3c) [reply ru/building-sites/client-proofing/security/policies/policytemplates.md]: Внёс правки

File: ru/building-sites/client-proofing/security/policies/permissions/resource-policy.md (hunk 2 above)
Comment (elec3c) [reply ru/building-sites/client-proofing/security/policies/permissions/resource-policy.md]: Внёс правки

**Comment by Ibochkarev** [comment]
@elec3c Thanks

**Pull request merged**